src/components/breadcrumb-bar/utils/get-docs-breadcrumbs.ts

```diff
@@ -105,14 +106,16 @@ function getPathMatchedNode(navNodes, pathString, basePath) {
   // Otherwise, we have zero matches, which would mean a breadcrumb with missing parts.
   // We have no nav-data to render a nice "title" for the breadcrumb bar, and
   // there isn't a matched path to link to, but we can still render an
-  // unlinked breadcrumb item using the "pathString" as the title
+  // unlinked breadcrumb item using fallback title text
   if (IS_DEV) {
     console.warn(
       `Missing breadcrumb path under "${basePath}": Found zero matches for "${pathString}". Please ensure there is a node (or index-less category) with the path "${pathString}" in the provided navData.`
     )
   }
-  const lastPathPart = pathString.split('/').pop()
-  return { title: lastPathPart }
+  // Get a fallback title by searching all navNodes for title text that,
+  // in slug form, matches the last path part. At worst, path will be used.
+  const fallbackTitle = getFallbackTitle(pathString, navNodes)
```

Comment:
`getFallbackTitle` is the approach I took initially, though it fits an even edgier case than I thought. It might not be that useful (the edge case may _never_ come up), so I've added a comment to flag that.